public/libs/views.js

```diff
@@ -194,8 +202,17 @@ const addView = (browserWindow, workspace) => {
     };
   }
 
-  view.webContents.on('will-navigate', (e, url) => {
-    adjustUserAgentByUrl(e.sender.webContents, url);
+  view.webContents.on('will-navigate', (e, nextUrl) => {
+    // open external links in browser
+    // https://github.com/atomery/webcatalog/issues/849#issuecomment-629587264
+    const appUrl = getWorkspace(workspace.id).homeUrl;
+    const currentUrl = e.sender.getURL();
+    if (!isInternalUrl(nextUrl, [appUrl, currentUrl])) {
+      e.preventDefault();
+      shell.openExternal(nextUrl);
```

Comment:
Not familiar with your codebase so disregard if this is irrelevant but I think you might be missing a return here (`adjustUserAgentByUrl` will still be called even when it's opening externally).

Reply:
Thank you so much for the review! I've fixed it.

https://github.com/atomery/singlebox/pull/266
https://github.com/atomery/juli/pull/13